### PR TITLE
ON-31968 # Fixed non element replaceable values not all being replaced

### PR DESCRIPTION
**Pull request by Zaxist**

### CODE REVIEW

File: src/replaceCustomValues.ts [hunk 1]
```diff
@@ -404,7 +404,7 @@ export function replaceInjectablesWithSubmissionValues(
   })
   return CUSTOM_VALUES.reduce((newString, customValue) => {
     return newString.replace(
```
Comment (mymattcarroll) [inline src/replaceCustomValues.ts:406]: Thoughts on using replaceAll here with the string?

Comment (Zaxist) [reply src/replaceCustomValues.ts:406]: Ahh, Thought that should have worked at first, but didn't come up as a function. Need to change compiler to target `es2021` if we want to use it.

Comment (Zaxist) [reply src/replaceCustomValues.ts:406]: Or later.

Comment (Zaxist) [reply src/replaceCustomValues.ts:406]: I haven't really thought too much about our target runtimes for SDKs. Prob not a drama if we want to update that.

Comment (mymattcarroll) [reply src/replaceCustomValues.ts:406]: nah, lets think about another time, merging!

**Pull request merged**